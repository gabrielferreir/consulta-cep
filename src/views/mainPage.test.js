import React from 'react';
import MainPage from './mainPage';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import {configure} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

it("Deve renderizar a pagina principal", () => {

    configure({adapter: new Adapter()});

    const app = shallow(<MainPage/>);

    expect(toJson(app)).toMatchSnapshot();
});