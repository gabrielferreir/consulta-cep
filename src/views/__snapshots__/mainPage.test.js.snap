// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar a pagina principal 1`] = `
<main
  className="main"
>
  <Connect(Header) />
  <Connect(Maps) />
</main>
`;
